### PR TITLE
Mismatched demographic data

### PR DESCRIPTION
**Pull request by susancglenn**
Updated participant_id_cleaning script to look for mismatched demographic data in the sex, position, and age group categories. I know age group will change for some individuals, I was just curious how much variation was happening in the group. I:
- created a dataframe that held relevant demographic information from all three timepoints, merged on participant_id_3
- added columns to that dataframe to identify if there was a mismatch in sex, position, or age group reported across three timepoints

I need to circle back to decide how to handle these mismatches. Cory made them NAs if they were misreported, but there are quite a few missing right now. I'm going to look more deeply at the data to decide how to proceed.

### CODE REVIEW

**Review by jessica-dyer — COMMENTED** [review]
Please update the code to the example I gave to identify mismatches in these demographics. It might be valuable to check the ones that don't match to see if maybe they are just incorrect matches from the match process. We can update the linklog to reflect that if needed. I'm not sure we should give the age too much weight as that can easily be given incorrectly or they actually have gotten a year older. I guess the position could have changed over the last almost two years, but sex should have remained the same.

File: code/participant_id_cleaning.R [hunk 1]
```diff
@@ -106,6 +106,40 @@ participant_id_table <-
     label = list(status ~ "Exclusive timepoint status", inclusive_status ~ "Inclusive timepoint status")
   )
 
+# Check for mismatched demographic data from pre, post-intensive, and follow-up tests
+dem_info <- clean_data %>%
+  select("participant_id_3", "time_point", "sex", "age", "position", "position_years_clean")
```
[the remark this answers is not in the document]
Comment (susancglenn) [reply code/participant_id_cleaning.R]: I updated the file to include the code you wrote above. Thank you! Will investigate the PIDs to check.

**Comment by susancglenn** [comment]
I created notes on each of the issues per PID in the attached file. Can we discuss Tuesday?
[link_log_notes.xlsx](https://github.com/GBV-Thesis-Research/gbv-health-provider-study/files/12392833/link_log_notes.xlsx)

**Review by jessica-dyer — APPROVED** [review]
Looks good! Thanks for cleaning this up.

**Pull request merged**